osquery/utils/aws/aws_util.cpp

```diff
@@ -646,4 +703,16 @@ void setAWSProxy(Aws::Client::ClientConfiguration& config) {
     config.proxyPassword = FLAGS_aws_proxy_password;
   }
 }
+
+void enableFIPSInClientConfig(const std::string& service,
+                              Aws::Client::ClientConfiguration& config) {
+  // Gov FIPS endpoints for Kinesis, EC2, STS are used as-is, do nothing
+  if (config.region.rfind("us-gov") == 0 &&
+      (service == "kinesis" || service == "ec2" || service == "sts")) {
+    return;
+  }
+
+  config.endpointOverride =
+      service + "-fips." + config.region + ".amazonaws.com";
+}
```

Comment:
What do you mean without the region checks?

If you mean not checking that it's a valid region to enable FIPS in, I don't think it's ideal to have osquery attempt to connect to some bogus url, even if the domain is from Amazon, it would raise red flags.

Reply:
Feels like a lot of not great options.

Either we guess URLs and then attempt to use them. Or we hardcode regions.

I think that of those two choices, hardcoding feels better. I doubt those regions will change quickly.